docs/API.md

```diff
@@ -728,6 +728,18 @@ const Counter = componentFromStream(props$ => {
 })
 ```
 
+### `mapPropsStreamWithConfig()`
```

Comment:
I know, now we will have an alternative solution for libraries, but I see no bad to provide defaults for an apps in an easy way.